webcomponents/tool/src/main/frontend/js/sakai-permissions.js

```diff
@@ -51,6 +51,7 @@ class SakaiPermissions extends SakaiElement {
 
     return {
       tool: String,
+      bundlekey: String,
```

Comment:
This will end up as bundleKey: {attribute: "bundle-key", type: String} if you use hyphenated attribute names